tests/Microsoft.Bot.Builder.AI.QnA.Tests/QnAMakerTests.cs

```diff
@@ -57,6 +57,38 @@ await CreateFlow(rootDialog)
             .StartTestAsync();
         }
 
+        [TestMethod]
+        public async Task QnAMakerAction_Test1()
```

Comment:
Please give a meaningful name to the test. If I see a failure in the test, I should understand what exact scenario and class are broken.

An example of naming would be

<thing to test>_When<condition / context>_Then<expected result>

Reply:
Done